raiden/network/proxies/payment_channel.py

```diff
@@ -283,6 +284,8 @@ def unlock(
 
     def settle(
         self,
+        raiden: "RaidenService",
+        channel_settle_event: ContractSendChannelSettle,
```

Comment:
why we need to modify the `settle `function and receive the `channel_settle_event`?